modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule.js

```diff
@@ -506,6 +508,32 @@ AUI.add(
 						instance._validateRule();
 					},
 
+					_handleFormBuildClick: function() {
```

Comment:
Can you please rename to _handleFormBuild**er**Click, please?

Reply:
Ok